src/cpp/server/DBActiveSessionStorage.cpp

```diff
@@ -0,0 +1,436 @@
+/*
+ * DBActiveSessionStorage.cpp
+ *
+ * Copyright (C) 2022 by RStudio, PBC
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+#include "server/DBActiveSessionStorage.hpp"
+#include "server_core/ServerDatabase.hpp"
+#include "core/Database.hpp"
+#include <numeric>
```

Comment:
A general rule of thumb for organising includes (plus some other include things):

* The `hpp` associated with this `cpp` goes first (as you have it)
* Other includes should be separated from it by a blank line
* Includes can be blocked together in groups based on the include location (e.g. all `<core/*>` includes together and all `<boost/*>` together, with newlines between) although for this case, it's probably not necessary to split them up because there are so few
* We should use `<>` instead of `""` wherever possible. `""` will include the current directory in the search path whereas `<>` will not.
* Includes from the current directory (requiring `""`) should go last, in their own block
* I like to organise things within a block alphabetically, but that's purely personal preference

```suggestion
#include <server/DBActiveSessionStorage.hpp>

#include <core/Database.hpp>
#include <server_core/ServerDatabase.hpp>

#include <numeric>
```